packages/ebayui-core/src/components/ebay-filter-chip/component.ts

```diff
@@ -0,0 +1,66 @@
+import type { AttrString } from "marko/tags-html";
+import type { WithNormalizedProps } from "../../global";
+import type { AttrBoolean } from "marko/tags-html";
+
+interface Image extends Marko.HTML.Img {
+    as?: string;
+}
+
+interface FilterChipInput extends Omit<Marko.HTML.Button, `on${string}` | "type">,
+        Omit<Marko.HTML.A, `on${string}`> {
+    renderBody?: Marko.Body;
+    selected?: boolean;
+    variant?: "default" | "expressive" | "menu";
+    disabled?: AttrBoolean;
+    icon?: Marko.AttrTag<{ renderBody?: Marko.Body }>;
+    image?: Marko.AttrTag<Image>;
+    clippedText?: AttrString;
+    expanded?: boolean;
+    "on-click"?: (event: {
+        originalEvent: MouseEvent;
+        expanded: boolean;
+        selected: boolean;
+    }) => void;
+}
+export interface Input extends WithNormalizedProps<FilterChipInput> {}
+
+export interface State {
+    expanded?: boolean;
+    selected?: boolean;
+    loaded: boolean;
```

Comment:
Should we use `mounted` instead of `loaded` so it is aligned with chip-input state name also?